main.go

```diff
@@ -1,21 +1,73 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"os"
+	"os/user"
+	"path/filepath"
 
 	"github.com/abs-lang/abs/repl"
+	"github.com/hashicorp/go-getter"
 )
 
 var Version = "1.8.0"
 
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
 // The ABS interpreter
 func main() {
 	args := os.Args
 	if len(args) == 2 && args[1] == "--version" {
 		fmt.Println(Version)
 		return
 	}
+
+	if args[1] == "--get" {
+		if len(args) < 3 {
+			fmt.Println("No module given")
+			return
+		}
+		dir := fmt.Sprintf("./vendor/%s", args[2])
+		module := args[2]
+		err := getter.GetAny(dir, module)
+
+		u, _ := user.Current()
+		str, _ := getter.Detect(module, u.HomeDir, getter.Detectors)
+		fmt.Println("detected", str)
+		if err != nil {
+			fmt.Printf("Could not get module: %s\n", err)
+			return
+		}
+
+		f, err := os.OpenFile("./alias", os.O_RDWR|os.O_CREATE, 0666)
+		check(err)
+		defer f.Close()
+
+		b, err := ioutil.ReadAll(f)
+		data := make(map[string]string)
+		err = json.Unmarshal(b, &data)
+
+		base := filepath.Base(module)
+
+		// add alias pairs to file
+		data[base] = module
+
+		newData, err := json.MarshalIndent(data, "", "    ")
+		check(err)
+
+		_, err = f.WriteAt(newData, 0)
```

Comment:
If 2 modules with the sam name get downloaded, we should warn the user. say `mingwho/x` and `alex/x`. We can download the first one and have `x => mingwho/x` but when the second one gets downloaded we should alias it to `alex/x => alex/x` (so no alias really...) and tell the user that this module could not be aliased.

Also, it would be nice to print an helpful message to the user post module install such as:

```
You can use the installed module with "require('alias_of_this_module')"
```